backend/src/main/java/com/soen341/instagram/service/impl/CommentService.java

```diff
@@ -144,8 +114,14 @@ private Picture findPicture(final long pictureId)
 		return pictureOptional.get();
 	}
 
-	public Comment findComment(final long commentId)
+	public Comment findComment(final String id)
 	{
+		long commentId;
```

Comment:
Indentation here is wrong

Reply:
could you elaborate? I dont see anything wrong with the indentation
![image](https://user-images.githubusercontent.com/45129406/76909206-fe504f00-6880-11ea-94f7-5ad2e0d00ad9.png)